### PR TITLE
Missing prereq (node-gyp) and update links

### PR DESCRIPTION
**Pull request by amychan331**
@go-bears 

1. Error message occurs on systems that don't have node-gyp installed and node-sass wouldn't install correctly as a result, so it's best to warn users beforehand.
2. Some of the links in README.md is absolute links to the old hack4impact-uiuc repo, so I update those to relative links instead.

...I just realize my Sublime Text (I think?) have removed the trailing whitespace automatically, which is why there's a lot more red line than I expected. Let me know if you want the whitespace back in.

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -2,11 +2,15 @@
 Clone the kiva-pricing-tool repository onto your computer.
 
 ## Frontend
+If you don't node-gyp installed globally, do so now
```
Comment (go-bears) [inline docs/setup.md]: Maybe you'd like to change the line to  "if you don't have"? @amychan331

**Comment by go-bears** [comment]
Thanks! @amychan331 !  Congrats on your first pull request on Kiva's open source repo!

File: docs/setup.md (hunk 1 above)
Comment (amychan331) [reply docs/setup.md]: Oops... missing word now fixed!

**Comment by go-bears** [comment]
@coderintherye  This PR looks good to merge. Thanks so much @amychan331 ! & congrats on your first pull request for Kiva!

**Comment by coderintherye** [comment]
+1 thanks @amychan331

**Pull request merged**